wled00/wled.h

```diff
@@ -105,6 +107,9 @@
 #endif
 #include <WiFiUdp.h>
 #include <DNSServer.h>
+#ifndef WLED_DISABLE_REMOTE
+  #include <espnow.h>
+#endif
```

Comment:
Seems that esp32 needs a different header

https://github.com/espressif/arduino-esp32/blob/1.0.6/libraries/ESP32/examples/ESPNow/Basic/Master/Master.ino

```c++
#include <esp_now.h>
```
You could include the correct header for each platform this way: 
```c++
#ifdef ARDUINO_ARCH_ESP32
  #include <esp_now.h>
#else
  #include <espnow.h>
#endif

```